Delivered-To: koen@dominion.thruhere.net
Received: by 10.236.157.72 with SMTP id n48cs4581yhk;
        Fri, 5 Aug 2011 00:10:45 -0700 (PDT)
Received: by 10.147.146.12 with SMTP id y12mr1585882yan.12.1312528245335;
        Fri, 05 Aug 2011 00:10:45 -0700 (PDT)
Return-Path: <archit@ti.com>
Received: from bear.ext.ti.com (bear.ext.ti.com [192.94.94.41])
        by mx.google.com with ESMTPS id x20si3422127ani.205.2011.08.05.00.10.45
        (version=TLSv1/SSLv3 cipher=OTHER);
        Fri, 05 Aug 2011 00:10:45 -0700 (PDT)
Received-SPF: pass (google.com: domain of archit@ti.com designates 192.94.94.41 as permitted sender) client-ip=192.94.94.41;
Authentication-Results: mx.google.com; spf=pass (google.com: domain of archit@ti.com designates 192.94.94.41 as permitted sender) smtp.mail=archit@ti.com
Received: from dlep33.itg.ti.com ([157.170.170.112])
	by bear.ext.ti.com (8.13.7/8.13.7) with ESMTP id p757AiHb023389
	(version=TLSv1/SSLv3 cipher=DHE-RSA-AES256-SHA bits=256 verify=NO);
	Fri, 5 Aug 2011 02:10:44 -0500
Received: from dlep26.itg.ti.com (smtp-le.itg.ti.com [157.170.170.27])
	by dlep33.itg.ti.com (8.13.7/8.13.8) with ESMTP id p757AiWp004418;
	Fri, 5 Aug 2011 02:10:44 -0500 (CDT)
Received: from dlee73.ent.ti.com (localhost [127.0.0.1])
	by dlep26.itg.ti.com (8.13.8/8.13.8) with ESMTP id p757AiBJ012377;
	Fri, 5 Aug 2011 02:10:44 -0500 (CDT)
Received: from dlelxv23.itg.ti.com (172.17.1.198) by DLEE73.ent.ti.com
 (157.170.170.88) with Microsoft SMTP Server id 8.3.106.1; Fri, 5 Aug 2011
 02:10:44 -0500
Received: from legion.dal.design.ti.com (legion.dal.design.ti.com
 [128.247.22.53])	by dlelxv23.itg.ti.com (8.13.8/8.13.8) with ESMTP id
 p757Ai9V018852;	Fri, 5 Aug 2011 02:10:44 -0500
Received: from localhost (a0393947pc.apr.dhcp.ti.com [172.24.137.144])	by
 legion.dal.design.ti.com (8.11.7p1+Sun/8.11.7) with ESMTP id p757Aef10060;
	Fri, 5 Aug 2011 02:10:41 -0500 (CDT)
From: Archit Taneja <archit@ti.com>
To: <hvaibhav@ti.com>, <linux-media@vger.kernel.org>
CC: <koen@dominion.thruhere.net>, <tomi.valkeinen@ti.com>,
        <linux-omap@vger.kernel.org>, Archit Taneja <archit@ti.com>
Subject: [PATCH] [media] OMAP_VOUT: Fix build break caused by update_mode removal in DSS2
Date: Fri, 5 Aug 2011 12:49:21 +0530
Message-ID: <1312528761-18241-1-git-send-email-archit@ti.com>
X-Mailer: git-send-email 1.7.1
MIME-Version: 1.0
Content-Type: text/plain

The DSS2 driver does not support the configuration of the update_mode of a
panel anymore. Remove the setting of update_mode done in omap_vout_probe().
Ignore configuration of TE since omap_vout driver doesn't support manual update
displays anyway.

Signed-off-by: Archit Taneja <archit@ti.com>
---
 drivers/media/video/omap/omap_vout.c |   13 -------------
 1 files changed, 0 insertions(+), 13 deletions(-)

diff --git a/drivers/media/video/omap/omap_vout.c b/drivers/media/video/omap/omap_vout.c
index b5ef362..b3a5ecd 100644
--- a/drivers/media/video/omap/omap_vout.c
+++ b/drivers/media/video/omap/omap_vout.c
@@ -2194,19 +2194,6 @@ static int __init omap_vout_probe(struct platform_device *pdev)
 					"'%s' Display already enabled\n",
 					def_display->name);
 			}
-			/* set the update mode */
-			if (def_display->caps &
-					OMAP_DSS_DISPLAY_CAP_MANUAL_UPDATE) {
-				if (dssdrv->enable_te)
-					dssdrv->enable_te(def_display, 0);
-				if (dssdrv->set_update_mode)
-					dssdrv->set_update_mode(def_display,
-							OMAP_DSS_UPDATE_MANUAL);
-			} else {
-				if (dssdrv->set_update_mode)
-					dssdrv->set_update_mode(def_display,
-							OMAP_DSS_UPDATE_AUTO);
-			}
 		}
 	}
 
-- 
1.7.1

